moduleroot/.travis.yml.erb

```diff
@@ -131,6 +131,28 @@ jobs:
 <%   end -%>
 <%  end -%>
 <% end -%>
+<% if @configs['use_litmus_complex']
+  @configs['litmus']['complex'].each do |config|
```

Comment:
Duplicating this whole section will inevitably lead to someone changing one and not the other. It should be possible to collapse this into `(@configs['litmus']['complex'] + [{'collection' =>@configs['litmus'] }]).each do |config|` to have a loop over both the `complex` and the regular config.

Reply:
It might be possible? But it might cause a problem doing that if someone where to give input for the more complex one but leave the standard one blank, i.e.:
```
  use_litmus: true
    provision_list:
      - ---travis_deb
      - ---travis_el
    complex:
    - collection:
        puppet_collection:
        - puppet6
        provision_list:
        - travis_ub_6
    - collection:
        puppet_collection:
        - puppet5
        provision_list:
        - travis_ub_5
```

Reply:
nvm, I can just put in a null check. will try now

Reply:
Code is updated with suggested change.

Reply:
hmm - is there ever a reason to have the standard provision_list values if you specify complex?

Reply:
As it is `use_litmus_complex` is no longer specified, the code simply detects if a complex value is given, as long as `use_litmus` is specified.
I had left the standard provisioning in in order to maintain functionality and not make it backwards incompatible.
Didn't want to inconvenience anyone I didn't need to.

Reply:
As it is `use_litmus_complex` is no longer specified, the code simply detects if a complex value is given, as long as `use_litmus` is specified.
I had left the standard provisioning in in order to maintain functionality and not make it backwards incompatible.
Didn't want to inconvenience anyone I didn't need to.